### PR TITLE
[EYPP-173] Add CLI for environment variables.

### PR DESCRIPTION
**Pull request by rzaharenkov**

### CODE REVIEW

**Comment by ehowe** [comment]
Looks good but I'm going to hold off on merging/releasing until the awsm stuff is merged and released

**Comment by jacobo** [comment]
needs ability to set and delete environment variables

File: lib/ey-core/models/environment_variable.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  has_one :environment
+  has_one :application
+
+  def save!
+    requires :name, :value
+    params = Cistern::Hash.slice(Cistern::Hash.stringify_keys(attributes), "id", "name", "value")
+    if new_record?
+      requires :application_id, :environment_id
+      params.merge!("application_id" => application_id, "environment_id" => environment_id)
+      merge_attributes(self.connection.create_environment_variable(params).body["environment_variable"])
```
Comment (ddiachkov) [inline lib/ey-core/models/environment_variable.rb]: Duplicate code. You can remove this one and move out of else next second copy of it.

**Comment by ddiachkov** [comment]
Also I don't see destroy variable request.

File: lib/ey-core/cli/environment_variables.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+require 'ey-core/cli/subcommand'
+require 'ey-core/cli/helpers/stream_printer'
+
+module Ey
+  module Core
+    module Cli
+      class EnvironmentVariables < Subcommand
+        include Ey::Core::Cli::Helpers::StreamPrinter
+        title "environment_variables"
```
[the remark this answers is not in the document]
Comment (rzaharenkov) [reply lib/ey-core/cli/environment_variables.rb]: It is available with `ey console` we never provide CRUD via CLI.

**Comment by ddiachkov** [comment]
LGTM. But I am not gonna merge it until we deploy ALB and autoscaling.

**Pull request merged**